www/~/%username/emails/modify.json.spt

```diff
@@ -46,7 +47,10 @@ if action in ('add-email', 'resend', 'start-verification'):
         raise Response(400)
 
     participant.start_email_verification(address, *packages)
-    msg = _("Check your inbox for a verification link.")
+    if show_address_in_message:
```

Comment:
This is confusing 😕 `verification.spt` is used both to verify emails and to verify linking packages to emails. The pattern of `if` conditions suggests that there isn't much in common either - I think it is best to separate these out.